airflow/executors/celery_executor.py

```diff
@@ -57,17 +61,17 @@
 
 
 @app.task
-def execute_command(command_to_exec):
+def task(command_to_exec: str):
+    """Executes command."""
     log = LoggingMixin().log
     log.info("Executing command in Celery: %s", command_to_exec)
     env = os.environ.copy()
     try:
         subprocess.check_call(command_to_exec, stderr=subprocess.STDOUT,
                               close_fds=True, env=env)
     except subprocess.CalledProcessError as e:
-        log.exception('execute_command encountered a CalledProcessError')
+        log.exception('execute command encountered a CalledProcessError')
```

Comment:
We should change this exception. As `execute_command` is renamed to `task` (Maybe we should change the name from task to something else to not confuse with Airflow's task). WDYT?

```diff
- log.exception('execute command encountered a CalledProcessError')
+ log.exception('task encountered a CalledProcessError')
```

Reply:
Yeah. I changed it back to execute_command :)

Reply:
```suggestion
        log.exception('execute_command encountered a CalledProcessError')
```

Reply:
^^